### DISCUSSION TITLE
Welcome to supabase-ssr-starter Discussions!

### DISCUSSION THREAD

**Discussion by nermalcat69** [nermalcat69/supabase-ssr-starter#2] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by SusheelThapa** [comment]
```
Name: Susheel Thapa
Address: Nepal, South Asia
Tech Stack: MERN, Django with MySQL/PostgreSQL
```
  
    I would like to build a network of tech enthusiast from this community. Also, I am looking forward to learn for them.

### **Connect With Me:**
[Github](https://github.com/SusheelThapa)
[Linkedin](https://www.linkedin.com/in/susheelthapa/)

**Comment by huilensolis** [comment]
Hi, I would like to contribute to the project. do you have any feature ideas? if so, create some issues so we can contribute : )

**Reply by nermalcat69** [reply]
yeah sure 😸! will do it and @ you on the issue
